src/hooks/test/use-click-away-test.js

```diff
@@ -62,7 +60,23 @@ describe('useClickAway', () => {
       });
       wrapper.update();
 
-      assert.equal(handler.callCount, 0);
+      assert.notCalled(handler);
+    });
+
+    it(`should not invoke callback on elements removed from container (${event.type})`, () => {
```

Comment:
```suggestion
    it(`should not invoke callback if clicked element is removed from container (${event.type})`, () => {
```

The previous wording to me suggested the callback being invoked with the element as an argument.